### PR TITLE
docs(k8s): add monitoring docs

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Check that the commit messages match our requested structure.
- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: containers/kubernetes/how-to/access-kubernetes-dashboard.mdx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+</Message>
+
+## How to access the Kubernetes dashboard
+
+1. Click **Kubernetes** in the **Containers** section of the [Scaleway Console](https://console.scaleway.com) side menu. The Kubernetes Kapsule dashboard displays.
```
Comment (farhakennar) [inline containers/kubernetes/how-to/access-kubernetes-dashboard.mdx:25]:
```suggestion
1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com) side menu. The Kubernetes Kapsule dashboard displays.
```

File: containers/kubernetes/how-to/access-kubernetes-dashboard.mdx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+2. Click the name of the cluster you want to access. The cluster overview page displays.  This page provides several pieces of information about your cluster:
```
Comment (farhakennar) [inline containers/kubernetes/how-to/access-kubernetes-dashboard.mdx:26]:
```suggestion
2. Click the name of the cluster you want to access. The cluster overview page displays. This page provides several pieces of information about your cluster:
```

File: containers/kubernetes/how-to/monitor-cluster.mdx [hunk 3]
```diff
@@ -18,27 +18,45 @@ categories:
...
 
-If you click the **Pools** tab, you are able to add, edit or delete a pool on your cluster.
+<Message type="important">
+  You must have [activated](/observability/cockpit/how-to/activate-cockpit/) your Cockpit. Cockpit’s free plan retention time being 31 days for metrics and 7 days for logs.
+</Message>
+
+1. Open your Cockpit in the [Scaleway console](https://console.scaleway.com/cockpit/overview).
+2. Click **Manage Scaleway data**. The Grafana interface opens in a new browser tab.
+3. Log into Grafana using your Cockpits' Grafana user and password. The list of your Cockpit dashboards displays.
+4. Click **Kubernetes Cluster Overview** in the list of dashboards. A status overview of your cluster displays. This dashboard allows you to monitor multiple components of your Kubernetes cluster: control-plane, nodes, managed resources and the cluster system applications.
```
Comment (farhakennar) [inline containers/kubernetes/how-to/monitor-cluster.mdx:41]:
```suggestion
4. Click **Kubernetes Cluster Overview** in the list of dashboards. A status overview of your cluster displays. This dashboard allows you to monitor multiple components of your Kubernetes cluster: control-plane, nodes, managed resources and cluster system applications.
```

**Review by farhakennar — DISMISSED** [review]

File: containers/kubernetes/how-to/monitor-cluster.mdx (hunk 3 above)
Comment (farhakennar) [inline containers/kubernetes/how-to/monitor-cluster.mdx:41]:
```suggestion
4. Click **Kubernetes Cluster Overview** in the list of dashboards. A status overview of your cluster displays. This dashboard allows you to monitor multiple components of your Kubernetes cluster: control-plane, nodes, managed resources and cluster system applications.
```

File: containers/kubernetes/how-to/monitor-cluster.mdx [hunk 4]
```diff
@@ -18,27 +18,57 @@ categories:
...
-2. Click the name of the cluster you want to monitor. The cluster overview page displays.  This page provides several pieces of information about your cluster:
-    * Cluster information
-    * Auto Upgrade
-    * Cluster tags
-    * Renewal of the kubeconfig file to restore access permissions
-    * Download of the kubeconfig file
-    * The option to delete the cluster
+[Cockpit](/observability/cockpit/quickstart/) empowers you to monitor the clusters' control plane, nodes, managed resources and the cluster system applications.
+* Detect incidents and diagnose failures in the earliest stages
+* Get new insights about control plane and worker nodes usage such as CPU, memory, to optimize your resource allocation
```
Comment (nerda-codes) [inline containers/kubernetes/how-to/monitor-cluster.mdx:29]:
```suggestion
* Get new insights on control plane and worker nodes usage such as CPU and memory to optimize your resource allocation
```

File: containers/kubernetes/how-to/monitor-cluster.mdx [hunk 5]
```diff
@@ -18,27 +18,57 @@ categories:
...
+* Achieve a better understanding of your infrastructure by analyzing data on your API server, system applications, up and down scaling behavior
 
-If you click the **Dashboard** button, you are taken to your cluster's Kubernetes dashboard. Here you can troubleshoot your containerized application, manage the cluster resources, get an overview of applications running on your cluster, and create or modify certain resources.
+## How to access your Cockpit Grafana dashboard
 
-If you click the **Pools** tab, you are able to add, edit or delete a pool on your cluster.
+<Message type="important">
+  You must have [activated](/observability/cockpit/how-to/activate-cockpit/) your Cockpit. Cockpit’s free plan retention time being 31 days for metrics and 7 days for logs.
```
Comment (nerda-codes) [inline containers/kubernetes/how-to/monitor-cluster.mdx:35]:
```suggestion
  You must have [activated](/observability/cockpit/how-to/activate-cockpit/) your Cockpit. Cockpit’s free plan's retention time is 31 days for metrics and 7 days for logs.
```

File: containers/kubernetes/how-to/monitor-cluster.mdx [hunk 6]
```diff
@@ -18,27 +18,57 @@ categories:
...
+</Message>
+
+1. Open your Cockpit in the [Scaleway console](https://console.scaleway.com/cockpit/overview).
```
Comment (nerda-codes) [inline containers/kubernetes/how-to/monitor-cluster.mdx:38]: There should be a step after this one to create a Grafana user, so they are able to log in to Grafana with their credentials or maybe add it to the requirements

File: containers/kubernetes/how-to/monitor-cluster.mdx [hunk 7]
```diff
@@ -18,27 +18,57 @@ categories:
...
+    <Lightbox src="scaleway-k8s-cluster-metrics.webp" alt="" />
+    <Message type="tip">
+      Select the cluster you want to display from the drop-down list at the top of the page to change the currently displayed cluster.
+    </Message>
+
+### Monitoring of the kube-apiserver
+
+The cluster overview dashboard offers real-time monitoring capabilities for the `kube-apiserver` within your Kubernetes cluster. Serving as a crucial component on the control plane, the `kube-apiserver` acts as the gateway to the Kubernetes API, allowing you to interact with the cluster.
+
+In cases where your clusters are large and containing numerous resources such as nodes, pods, and custom resources (CRDs), or when experiencing a large number of simultaneous requests to the apiserver from controllers like argocd and velero,  it is possible for the CPU and memory usage to spike. Consequently, the performance of the apiserver may noticeably decline, leading to sluggishness or unresponsiveness. You might encounter errors like `"EOF"` when using `kubectl`.
```
Comment (nerda-codes) [inline containers/kubernetes/how-to/monitor-cluster.mdx:51]: This sentence is a bit long, maybe try making it shorter?

Comment (nerda-codes) [inline containers/kubernetes/how-to/monitor-cluster.mdx:51]:
```suggestion
In cases where your clusters are large and contain many resources (nodes, pods, and custom resources (CRDs)), or when experiencing a large number of simultaneous requests to the apiserver from controllers (such as  argocd and velero),  it is possible for the CPU and memory usage to spike. Consequently, the performance of the apiserver may noticeably decline, leading to sluggishness or unresponsiveness. You might encounter errors like `"EOF"` when using `kubectl`.
```

Comment (bene2k1) [reply containers/kubernetes/how-to/monitor-cluster.mdx]: Can't make it shorter. If you have any idea how, let me know

Comment (nerda-codes) [reply containers/kubernetes/how-to/monitor-cluster.mdx]: The CPU and memory usage might spike for large clusters containing many resources (nodes, pods, and custom resources) or experiencing many simultaneous requests to the API server from controllers such as argocd and velero. 

Something like that maybe ?

**Review by nerda-codes — APPROVED** [review]

**Comment by tgenaitay** [comment]
LGTM! Thanks!

File: containers/kubernetes/how-to/access-kubernetes-dashboard.mdx [hunk 8]
```diff
@@ -0,0 +1,44 @@
+---
+meta:
+  title: How to access the dashboard of a Kubernetes Kapsule cluster
+  description: This page explains how to access the dashboard of your Kubernetes Kapsule cluster
+content:
+  h1: How to access the dashboard of a Kubernetes Kapsule cluster
+  paragraph: This page explains how to access the dashboard of your Kubernetes Kapsule cluster
+tags: kubernetes kubernetes-kapsule kapsule cluster 
+dates:
+  validation: 2023-02-07
```
Comment (RoRoJ) [inline containers/kubernetes/how-to/access-kubernetes-dashboard.mdx:10]: dates need changing?

File: containers/kubernetes/how-to/access-kubernetes-dashboard.mdx [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+  - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+</Message>
+
+## How to access the Kubernetes dashboard
+
+1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com) side menu. The Kubernetes Kapsule dashboard displays.
```
Comment (RoRoJ) [inline containers/kubernetes/how-to/access-kubernetes-dashboard.mdx:25]: Maybe confusing that we talk about a Kubernetes dashboard here, but it's another type of dashboard!

File: containers/kubernetes/how-to/monitor-cluster.mdx [hunk 10]
```diff
@@ -1,44 +1,78 @@
...
 <Message type="requirement">
-  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
   - You have [created](/containers/kubernetes/how-to/create-cluster) a Kubernetes Kapsule cluster
+  - You have [activated](/observability/cockpit/how-to/activate-cockpit/) your observability Cockpit
+  - You have [created](/observability/cockpit/how-to/retrieve-grafana-credentials/) a Grafana user for Cockpit
 </Message>
 
-## How to access the Kubernetes dashboard
+Scaleway's observability Cockpit provides automated monitoring of your Kubernetes cluster. As Kubernetes environments grow in complexity, monitoring becomes crucial for optimal performance. Visualization of the clusters' metrics are available in a hosted Grafana.
```
Comment (RoRoJ) [inline containers/kubernetes/how-to/monitor-cluster.mdx:25]:
```suggestion
Scaleway's observability Cockpit provides automated monitoring of your Kubernetes cluster. As Kubernetes environments grow in complexity, monitoring becomes crucial for optimal performance. Visualization of the clusters' metrics is available in a hosted Grafana.
```

File: containers/kubernetes/how-to/access-kubernetes-dashboard.mdx (hunk 8 above)
Comment (bene2k1) [inline containers/kubernetes/how-to/access-kubernetes-dashboard.mdx:10]:
```suggestion
  validation: 2023-08-08
```

File: containers/kubernetes/how-to/access-kubernetes-dashboard.mdx (hunk 9 above)
Comment (bene2k1) [inline containers/kubernetes/how-to/access-kubernetes-dashboard.mdx:25]:
```suggestion
1. Click **Kubernetes** in the **Containers** section of the [Scaleway console](https://console.scaleway.com) side menu. The Kubernetes Kapsule overview displays.
```

File: containers/kubernetes/how-to/monitor-cluster.mdx [hunk 11]
```diff
@@ -1,44 +1,78 @@
...
-  h1: How to monitor your Kubernetes Kapsule cluster
-  paragraph: This page explains how to monitor your Kubernetes Kapsule cluster
-tags: kubernetes kubernetes-kapsule kapsule cluster 
+  h1: How to monitor your Kubernetes Kapsule cluster with Cockpit
+  paragraph: This page explains how to monitor your Kubernetes Kapsule cluster with Cockpit
+tags: kubernetes kubernetes-kapsule kapsule cluster observability cockpit
 dates:
-  validation: 2023-02-07
-  posted: 2020-10-14
+  validation: 2023-07-24
```
Comment (bene2k1) [inline containers/kubernetes/how-to/monitor-cluster.mdx:10]:
```suggestion
  validation: 2023-08-08
```

**Review by RoRoJ — APPROVED** [review]

File: containers/kubernetes/how-to/monitor-cluster.mdx [hunk 12]
```diff
@@ -1,44 +1,78 @@
...
 
-If you click the **Nodes** tab, you are able to reboot or replace nodes on your cluster.
+## How to view the Kubernetes control plane logs
 
+1. Open your Cockpit in the [Scaleway console](https://console.scaleway.com/cockpit/overview).
+2. Click **Manage Scaleway data**. The Grafana interface opens in a new browser tab.
+3. Log into Grafana using your Cockpits' Grafana user and password. The list of your Cockpit dashboards displays.
+4. Click **Kubernetes Cluster Logs** in the list of dashboards. A listing of your control plane logs displays. This dashboard allows you to view the logs of multiple components of your Kubernetes cluster, such as the `controller-manager`, `ccm`, `kapsule-autoscaler`, and more.
+    <Message type="note">
+      Currently no logs from the `kube-apiserver` are sent to your Cockpit.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/monitor-cluster.mdx:68]:
```suggestion
      Currently, no logs from the `kube-apiserver` are sent to your Cockpit.
```

File: containers/kubernetes/how-to/monitor-cluster.mdx [hunk 13]
```diff
@@ -1,44 +1,78 @@
...
+
+### Monitoring of the kube-apiserver
+
+The cluster overview dashboard offers real-time monitoring capabilities for the `kube-apiserver` within your Kubernetes cluster. Serving as a crucial component on the control plane, the `kube-apiserver` acts as the gateway to the Kubernetes API, allowing you to interact with the cluster.
+
+Large clusters with numerous resources (nodes, pods, and custom resources (CRDs)) and high controller requests (e.g. argocd and velero) can cause CPU and memory spikes, leading to sluggish or unresponsive API server performance. You may also encounter errors like `"EOF"` when using `kubectl`.
+
+To address this potential issue, it is crucial to monitor the CPU and RAM consumption of the apiserver closely. By doing so, you can proactively manage and reduce the load on the apiserver, thus averting performance bottlenecks.
+
+For scenarios where you require consistent and predictable performance, along with dedicated resources for the apiserver, consider migrating it to a [dedicated contol plane](https://www.scaleway.com/en/kubernetes-dedicated-control-plane/) to ensure optimal performance and stability.
```
Comment (jcirinosclwy) [inline containers/kubernetes/how-to/monitor-cluster.mdx:59]:
```suggestion
For scenarios where you require consistent and predictable performance, along with dedicated resources for the apiserver, consider migrating it to a [dedicated control plane](https://www.scaleway.com/en/kubernetes-dedicated-control-plane/) to ensure optimal performance and stability.
```

**Pull request merged**